contracts/interfaces/IStakeRegistry.sol

```diff
@@ -42,4 +42,6 @@ interface IStakeRegistry {
     function createDToken(string calldata _name, string calldata _symbol, uint8 decimals)
         external
         returns (address);
+
+    function owner() external view returns (address);
```

Comment:
Shouldn't be necessary.

See `IMessenger(SLA(_sla).messengerAddress()).owner()` for an interface-based call of ownership for reference.